python/pyspark/mllib/stat/_statistics.py

```diff
@@ -65,11 +65,19 @@ def colStats(rdd):
         """
         Computes column-wise summary statistics for the input RDD[Vector].
 
-        :param rdd: an RDD[Vector] for which column-wise summary statistics
-                    are to be computed.
-        :return: :class:`MultivariateStatisticalSummary` object containing
-                 column-wise summary statistics.
-
+        Parameters
+        ----------
+        rdd : :py:class:`pyspark.RDD`
+            an RDD[Vector] for which column-wise summary statistics
```

Comment:
`RDD[Vector]` looks Scala syntax? How about RDD of Vector?

Reply:
Conveniently, that's syntax we use both for Scala and Python, with corresponding type hints looking like this:

https://github.com/apache/spark/blob/048a9821c788b6796d52d1e2a0cd174377ebd0f0/python/pyspark/mllib/stat/_statistics.pyi#L44